### PR TITLE
[RDY] Working status of game

### LINKED ISSUE #1884: Please describe the playable status in the Readme
**What feature or enhancement would you like to add?**
I love open source remakes of old games. However, very often they don't describe how playable the project is, and that doesn't allow users to create expectations and decide whether to use this OSS project or not. It seems this problem also applies to CorsixTH. I searched the [Readme](https://github.com/CorsixTH/CorsixTH/blob/master/README.md) and the [FAQ](https://github.com/CorsixTH/CorsixTH/wiki/Frequently-Asked-Questions) and I found no indication on CorsixTH's playable status. Some pieces about missing features can be gathered from [Programming Ideas](https://github.com/CorsixTH/CorsixTH/wiki/Programming-Ideas), however that is difficult to find and not fit for general consumption.

Please include a summary of the current playable status in the Readme (possibly linking to the FAQ where it can be expanded in greater depth). The playable status means "how playable in general this OSS remake is?", so e.g.:
* Is everything from the original game implemented, or is something missing?
* What are the notable features missing in the remake?
* Are there any high-level long-term bugs that people should be aware of?
* Are there any parts of the game missing (like the tutorial, campaign, multiplayer, etc)?
* Are there any important changes to the game or gameplay compared to the original, which should be mentioned (e.g. a different AI, tweaked difficulty, re-balanced economy, etc)? These changes don't need to be "better" or "worse", they are just different (e.g. because the original behavior was hard to replicate, so a different one was used).
* Are there any notable features which the remake has on top of the original (e.g. higher resolution support, larger maps support, an option to speed up time, etc)?

A section like this help users to decide whether to play CorsixTH or the original. (Often I worry that an incomplete OSS remake would make my experience worse, and because a status like this is not published and users are kept in the dark, I often end up playing the original instead. I'm sure there are more people like me). It would also help spread out the word and market your game, if the playable status is very good. And lastly it can attract some developers when they see that some pieces of the game are still waiting to be implemented.

Thanks :slightly_smiling_face:

### PR DESCRIPTION
**Pull request by lewri**
*Closes #1884*

**Describe what the proposed change does**
- Adds the game's working status to the readme as suggested.
- I tried to show a little humour again (it usually backfires though)

I don't think I wrote it in US English so point out any writing differences.

### CODE REVIEW

**Comment by lewri** [comment]
I realise this may be longer than what Cyco is generally after. However, my concern if putting it in the Wiki may cause it to decay/be neglected.

**Comment by TheCycoONE** [comment]
Don't worry about the English. This is largely a UK English game.

**Comment by TheCycoONE** [comment]
The one thing I would suggest is splitting what's working from the original from features the original never had.

File: README.md [hunk 1]
```diff
@@ -24,16 +24,21 @@ You will need the following:
 
 ### What's Working? ###
 Most features of the game are available -- and we're at a state where you can complete the full campaign without issue.
+##### Original Features #####
 - Single player modes (Campaigns, custom levels, custom campaigns)
```
Comment (TheCycoONE) [inline README.md:28]: The original game did not support custom levels or custom campaigns, those are BIG features of CorsixTH

File: README.md [hunk 2]
```diff
@@ -24,16 +24,21 @@ You will need the following:
...
 - All diseases, objects, rooms are available (see below section for outstanding anomalies)
 - All events (emergencies, earthquakes, epidemics, VIP visits)
 - Micromanagement
```
Comment (TheCycoONE) [inline README.md:31]: Micromanagement isn't clear?

File: README.md [hunk 3]
```diff
@@ -24,16 +24,21 @@ You will need the following:
...
+- Music/Jukebox and gameplay videos (see below section for oustanding anomalies)
+- Cheats (naughty!)
+##### New Features #####
 - Full HD resolution and beyond!
```
Comment (TheCycoONE) [inline README.md:35]: Our hidpi support is kinda poor, best to leave this at just widescreen or full HD support?

File: README.md [hunk 4]
```diff
@@ -24,16 +24,21 @@ You will need the following:
...
 - Make your own maps and levels
-- Music/Jukebox and gameplay videos (see below section for oustanding anomalies)
-   - You can also play your own music!
+- Unlimited saves and 12 autosave slots
+- Play your own music!
+- Option to remove destroyed rooms for a fee
+- Improved game logic
 - Full control over all hotkeys
```
Comment (TheCycoONE) [inline README.md:41]: Zoom is another big feature we have over the original, deserves mention.

**Review by TheCycoONE — CHANGES_REQUESTED** [review]

**Pull request merged**